src/index.js

```diff
@@ -0,0 +1,43 @@
+import currentLocation from './currentLocation';
+import render from './render';
+
+const searchBar = document.querySelector('.search-bar');
+const locationIcon = document.querySelector('.location-icon');
+
+const weather = (long, lat) => fetch(`http://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${long}&units=metric&APPID=4e1f41a544e8138e1382d0bff94cc988`, { mode: 'cors' })
+  .then(response => response.json())
+  .then((response) => {
+    render(response);
+  });
+
+const getCurrent = () => new Promise((resolve) => {
```

Comment:
I need clarification on this. Why does this function need to return a `Promise` ? At a high-level what is `getCurrent` supposed to achieve (single responsibility)?

Upon setting that single responsibility let's make sure that `getCurrent` just stick to doing that.